### DISCUSSION TITLE
Add descriptions and map names to screenshots on the website

### DISCUSSION THREAD

**Discussion by nok-ko** [redeclipse/base#1131] [Ideas and Feature Requests]
<img width="706" alt="screenshot_issue" src="https://user-images.githubusercontent.com/13428215/35773223-96119d36-0900-11e8-90c4-5b60af7ba122.png">
This is what happens once you click on one of the screenshots on our website.

Not much. I think that including a brief description of the situation, and the name of the map near the image would add a bit of quality to the page.

**Comment by IceflowRE** [comment]
I click on screenshots on the main page to get a better look at. Not to get informations, thats why theres a documentation: http://www.redeclipse.net/docs/Home or http://www.redeclipse.net/docs/Official-Maps. Anyway a link at the bottom to the map doc should be enough in my eyes.

**Comment by acerspyro** [comment]
The screenshots are rather old and need to be updated for Red Eclipse 2.0.

**Comment by nok-ko** [comment]
@IceflowRE, yeah, it doesn't have to be super-dense, a link to the map doc would suffice. 

@acerspyro, that would be nice.

**Comment by qreeves** [comment]
They're mostly there so people can see what the game looks like, rather much like the trailer.

Doing this would require maintaining something larger than just throwing screenshots into a folder, but it isn't outside the realm of possibility.
